.github/workflows/test_pr_changes.yml

```diff
@@ -20,6 +20,7 @@ jobs:
     # Set job outputs to values from filter step
     outputs:
       wire-ios: ${{ steps.filter.outputs.wire-ios == 'true' || steps.filter.outputs.carthage == 'true' }}
+      wire-user-profile: ${{ steps.filter.outputs.wire-user-profile }}
```

Comment:
is that a new package?

Reply:
A new target in WireUI.

Reply:
Do you prefer to combine all targets of a package and run the unit tests together?